app/models/asset_file.rb

```diff
@@ -8,7 +8,6 @@ module FlexCommerce
   # It is used much like an active record model.
   #
   class AssetFile < FlexCommerceApi::ApiBase
-    belongs_to :asset_folder
```

Comment:
This may be better switched to `has_one` as that allows the association to be optional.